python/GafferImageTest/FilterAlgoTest.py

```diff
@@ -121,8 +121,8 @@ def testFilterDerivatives( self ):
 
 		# Enable to write out images for visual comparison
 		if False:
-			IECore.Writer.create( parallelogramImage, "/tmp/filterDerivativesTestResult.parallelogram.exr" ).write()
-			IECore.Writer.create( boxImage, "/tmp/filterDerivativesTestResult.box.exr" ).write()
+			IECore.Writer.create( parallelogramImage, str( self.temporaryDirectory() / "filterDerivativesTestResult.parallelogram.exr" ) ).write()
+			IECore.Writer.create( boxImage, str( self.temporaryDirectory() / "filterDerivativesTestResult.box.exr" ) ).write()
```

Comment:
Although I don't know if `/tmp` exists on Windows or not, I don't think we want `self.temporaryDirectory()` here. That directory is deleted in `tearDown()`, so is useless for debugging. I'd be quite happy to leave as it was for now, or just delete this debug block - it's easy enough to stick one in when you need it.